src/main/java/com/microsoft/sqlserver/jdbc/PersistentTokenCacheAccessAspect.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Microsoft JDBC Driver for SQL Server Copyright(c) Microsoft Corporation All rights reserved. This program is made
+ * available under the terms of the MIT License. See the LICENSE file in the project root for more information.
+ */
+
+package com.microsoft.sqlserver.jdbc;
+
+import com.microsoft.aad.msal4j.ITokenCacheAccessAspect;
+import com.microsoft.aad.msal4j.ITokenCacheAccessContext;
+
+
+/**
+ * Access aspect for accessing the token cache.
+ * 
+ * MSAL token cache does not persist beyond lifetime of the application. This class implements the
+ * ITokenCacheAccessAspect interface to persist the token cache between application instances so subsequent
+ * authentications can use silent authentication if the user account is in the token cache.
+ * 
+ * @see <a href="https://aka.ms/msal4j-token-cache">https://aka.ms/msal4j-token-cache</a>
+ */
+class PersistentTokenCacheAccessAspect implements ITokenCacheAccessAspect {
+    private static PersistentTokenCacheAccessAspect instance = null;
```

Comment:
if this is changed to:
`private static PersistentTokenCacheAccessAspect instance = new PersistentTokenCacheAccessAspect();`
we can avoid a few null checks.